src/lib.rs

```diff
@@ -204,6 +204,14 @@ impl Arbitrary for char {
             }
         }
     }
+
+    fn shrink(&self) -> Box<dyn Iterator<Item = Self>> {
+        let x = *self as u32;
+        Box::new(x.shrink().filter_map(|x| {
+            use std::convert::TryFrom;
+            char::try_from(x).ok()
```

Comment:
this might be problematic for unicode correctness, it may abort early for large code points

Reply:
Hm I’m not sure how I’m familiar enough with Unicode to write this method properly. Do you have a suggested implementation or should I skip this method and drop this commit from the pull?

Reply:
Leave it for now, just make fixing this a part of #8